### DISCUSSION TITLE
https://github.com/organizations/XXXXX/settings/copilot/seat_management return 500 error

### DISCUSSION THREAD

**Discussion by lsyeastar** [community/community#55249] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Created a team named A, added it to Copilot, and then deleted it, resulting in this error.
We are currently unable to add Copilot members and cannot use Copilot. We hope to resolve this as soon as possible.

![image](https://github.com/community/community/assets/45458455/ce535d8d-86ca-4c37-a3f2-8892881a73ff)

![image](https://github.com/community/community/assets/45458455/daf843b4-83cd-4631-903c-dd46fe91beb7)

**Comment by Akash1134** [comment]
Hi @lsyeastar ,

This seems to be related to the [recent incidents](https://www.githubstatus.com/history) we've had that have now been resolved. 

Could you please try and add Copilot members to your Org and see if it works? 

Thanks!

**Comment by lsyeastar** [comment]
It's still the same error, and they have already started charging.
